src/main/java/com/googleinterns/zoomtube/data/Comment.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.auto.value.AutoValue;
+import java.util.Date;
+import javax.annotation.Nullable;
+
+/** Contains data related to a comment in a discussion. */
+@AutoValue
+public abstract class Comment {
+  public static final String ENTITY_KIND = "Comment";
+  public static final String PROP_LECTURE = "lecture";
+  public static final String PROP_PARENT = "parent";
+  public static final String PROP_TIMESTAMP = "timestamp";
+  public static final String PROP_AUTHOR = "author";
+  public static final String PROP_CONTENT = "content";
+  public static final String PROP_CREATED = "created";
+
+  public static Comment create(Key key, Key lecture, Key parent, double timestamp, String author,
+      String content, Date created) {
+    return new AutoValue_Comment(key, lecture, parent, timestamp, author, content, created);
+  }
+
+  public static Comment fromEntity(Entity entity) {
```

Comment:
should there be method headers for create() and fromEntity()?

Reply:
I think create is self-explanatory using the docs for the field names, but will add one for fromEntity()

Reply:
Done, thanks